### ISSUE TITLE
Documentation of the website

### ISSUE THREAD

**Issue by sabin-thapa** [kucc1997/kucc-website#38]
We need clean documentation for the website including all the essential features.

**Comment by erwinschrodinger1** [comment]
@sabin-thapa is the features as same as of kucc official site? If yes can you please assign this issue to me!

**Comment by sabin-thapa** [comment]
> @sabin-thapa is the features as same as of kucc official site? If yes can you please assign this issue to me!

Yes! More or less, it's the same. We are basically converting the wordpress site to React for now.
